IPython/utils/io.py

```diff
@@ -172,7 +172,7 @@ def ask_yes_no(prompt, default=None, interrupt=None):
     ans = None
     while ans not in answers.keys():
         try:
-            ans = input(prompt+' ').lower()
+            ans = builtins.input(prompt+' ').lower()
```

Comment:
I don't think we actually need `builtins` here - we can just use `input()` directly, I think.

Reply:
Yeah, good point. Fixed.